src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ManageContent/ManageContentPageServiceTests.cs

```diff
@@ -367,6 +355,146 @@ await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContex
+                    {
+                        Release = release,
+                        ContentSection = new()
+                        {
+                            Type = ContentSectionType.Headlines
+                        }
+                    },
+                    new()
+                    {
+                        Release = release,
+                        ContentSection = new()
+                        {
+                            Type = ContentSectionType.KeyStatisticsSecondary
+                        }
+                    },
+                    new()
+                    {
+                        Release = release,
+                        ContentSection = new()
+                        {
+                            Type = ContentSectionType.ReleaseSummary
+                        }
+                    },
+                    new()
+                    {
+                        Release = release,
+                        ContentSection = new()
+                        {
+                            Type = ContentSectionType.RelatedDashboards
+                        }
+                    },
+                    new()
+                    {
+                        Release = release,
+                        ContentSection = new()
+                        {
+                            Type = ContentSectionType.Generic,
+                        }
+                    });
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            var dataBlockService = new Mock<IDataBlockService>(MockBehavior.Strict);
+            var methodologyVersionRepository = new Mock<IMethodologyVersionRepository>(MockBehavior.Strict);
+            var releaseFileService = new Mock<IReleaseFileService>(MockBehavior.Strict);
+
+            dataBlockService.Setup(mock =>
+                    mock.GetUnattachedDataBlocks(release.Id))
+                .ReturnsAsync(new List<DataBlockViewModel>());
+
+            methodologyVersionRepository.Setup(mock =>
+                    mock.GetLatestPublishedVersionByPublication(publication.Id))
+                .ReturnsAsync(methodologies);
+
+            releaseFileService.Setup(mock =>
+                    mock.ListAll(release.Id, Ancillary, FileType.Data))
+                .ReturnsAsync(new List<FileInfo>());
+
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var service = SetupManageContentPageService(contentDbContext: contentDbContext,
+                    dataBlockService: dataBlockService.Object,
+                    methodologyVersionRepository: methodologyVersionRepository.Object,
+                    releaseFileService: releaseFileService.Object);
+
+                var result = await service.GetManageContentPageViewModel(
+                    release.Id, isPrerelease: true);
+
+                Assert.True(result.IsRight);
+
+                var contentRelease = result.Right.Release;
```

Comment:
You can use `var contentRelease = result.AssertRight()` here.

Reply:
I've removed all other `.Right`s from this test suite.